client/payment-details/dispute-details/style.scss

```
@@ -4,4 +4,11 @@
 	padding-left: 24px;
 	padding-right: 24px;
 	padding-bottom: 5px;
+
+	.transaction-details-dispute-details-body {
+		padding: 16px;
```

Small enhancement: looking at the design, I realise we can use grid-unit variables here.
```suggestion
		padding: $grid-unit-20;
```